system/doc/design_principles/distributed_applications.md

```diff
@@ -131,7 +131,19 @@ calling `application:start(myapp)` at all three nodes. It is then started at
 
 [](){: #dist1 }
 
-![Application myapp - Situation 1](assets/dist1.gif "Application myapp - Situation 1")
+```mermaid
```

Comment:
The original images here are not very good, but I think the new version looses important details, So please restore the distributed applications changes to their originals and we can tackle them later on.

Reply:
I personally felt that the new version did a good job of not loosing details compared to the supervisor diagrams. Perhaps if you specify what details are lost, I could tackle it myself (in a future commit)?